### DISCUSSION TITLE
readthedocs out of sync? - Compatibility questions

### DISCUSSION THREAD

**Discussion by Deathfireofdoom** [miguelgrinberg/python-socketio#1337] [Q&A] [answered]
At https://python-socketio.readthedocs.io/en/latest/intro.html there is a section about the work to support 3.X of socket-io. This section is not found here in github, and it seems the package already support 4.X.

Is the docs out of sync, or am I missing something?
<img width="685" alt="image" src="https://github.com/miguelgrinberg/python-socketio/assets/53213782/1a965a7b-6025-4fb3-9864-e885f9b143cc">

**Comment by miguelgrinberg** [comment, marked as answer]
Yes, looks like I forgot to update that section. The client and server are both compatible with Socket.IO 3.x and 4.x and have been for a long time. I'll update the docs, thanks!

**Reply by miguelgrinberg** [reply]
Docs are now updated. Thanks!
